lms/static/sass/shared/_footer-edx.scss

```diff
@@ -194,8 +422,223 @@ footer#footer-edx-v3 {
     .social-media-links {
       margin-bottom: 40px;
     }
+    .edx-col-md-12 {
```

Comment:
It is confusing to add additional rules to these grid classes. They are intended to only have one behavior which is to provide the grid. You should instead target another class, that is hopefully something more semantic.

Reply:
removed.